lib/twitter.js

```diff
@@ -0,0 +1,87 @@
+const querystring = require('querystring');
+const Twit = require('twit');
+const envVars = require('../env');
+
+const getClient = () => new Twit({
+  consumer_key: envVars.TWITTER_CONSUMER_KEY,
+  consumer_secret: envVars.TWITTER_CONSUMER_SECRET,
+  access_token: envVars.TWITTER_ACCESS_TOKEN,
+  access_token_secret: envVars.TWITTER_ACCESS_TOKEN_SECRET,
+});
+
+const sleep = (seconds = 5) => new Promise((resolve) => setTimeout(
+  resolve,
+  seconds * 1000,
+));
+
+const search = async (
+  client,
+  options,
+  iteration,
+  maxIterations,
+) => {
+  const thisIteration = iteration + 1;
+  const response = await client.get('search/tweets', options);
+  if (response.data && response.data.statuses) {
```

Comment:
This `if` block wraps almost an entire function. What about doing something like this?

```js
if (!response.data || !response.data.statuses) {
  return [];
}

// Rest of happy path follows...
```

That way you don't need to read the entire function to know its fallback case.